core/market/Cargo.toml

```diff
@@ -55,7 +55,7 @@ uuid = { version = "0.8", features = ["v4"] }
 
 # testing
 actix-http = { version = "2.2", optional = true }
-actix-rt = { version = "1.0.0", optional = true }
+actix-rt = { version = "1.0.0" }
```

Comment:
move it above pls, this is `#testing` section

Reply:
good catch! done